### PR TITLE
[RHPAM-2845] Expression in timer is not interpreted correctly

### PR DESCRIPTION
**Pull request by elguardian**
replace inverted commas in duration time expression for avoiding
java compilation problems.

### CODE REVIEW

**Comment by gmunozfe** [comment]
jenkins retest this

File: jbpm-bpmn2/src/test/resources/BPMN2-CallActivityWithBoundaryEvent.bpmn2 [hunk 1]
```diff
@@ -39,7 +39,7 @@
     <boundaryEvent id="BoundaryEvent_1" name="Boundary event" attachedToRef="_2">
       <outgoing>SequenceFlow_2</outgoing>
       <timerEventDefinition id="TimerEventDefinition_1">
-        <timeDuration xsi:type="tFormalExpression" id="_FormalExpression_2">2s</timeDuration>
+        <timeDuration xsi:type="tFormalExpression" id="_FormalExpression_2"><![CDATA[#{java.lang.System.getProperty("delayInProductionEnv","PT5s")}]]></timeDuration>
```
Comment (gmunozfe) [inline jbpm-bpmn2/src/test/resources/BPMN2-CallActivityWithBoundaryEvent.bpmn2:42]: is there any reason to extend it from 2s to 5s?

Comment (MarianMacik) [reply jbpm-bpmn2/src/test/resources/BPMN2-CallActivityWithBoundaryEvent.bpmn2:42]: +1, can we leave it as it was?

Comment (elguardian) [reply jbpm-bpmn2/src/test/resources/BPMN2-CallActivityWithBoundaryEvent.bpmn2]: done

**Comment by elguardian** [comment]
@MarianMacik @gmunozfe have a look. I did change the implementation but CI is a bit unstable.

**Comment by elguardian** [comment]
Jenkins retest this please.

**Comment by elguardian** [comment]
failure unrelated
Jenkins retest this please.

**Pull request merged**